cert_tools/helpers.py

```diff
@@ -57,5 +57,4 @@ def encode(num, alphabet=BASE62):
 
 
 def create_iso8601_tz():
-    ret = datetime.now(timezone.utc).isoformat()[:-13]+'Z'
-    return ret.isoformat()
\ No newline at end of file
+    return datetime.now(timezone.utc).isoformat()[:-13]+'Z'
```

Comment:
that looks good, now that I look at the other code, it was indeed a strange setup.